src/megaapi_impl.cpp

```diff
@@ -13509,40 +13555,57 @@ void MegaApiImpl::getua_result(TLVstore *tlv)
         request->setMegaStringMap(stringMap);
 
         // prepare request params to know if a warning should show or not
```

Comment:
I think the comment _// prepare request params to know if a warning should show or not_ should be just above the case sentence _case MegaApi::USER_ATTR_RICH_PREVIEWS:_